### ISSUE TITLE
Default values for Ontology Resource comment and Label

### ISSUE THREAD

**Issue by cstockloew** [universAAL/tools.eclipse-plugins#111]
_Originally Opened: Marina de la Fuente (2012-02-27 11:36:57_)
_Originally Closed: 2012-02-27 14:38:51_

All created ontologies have default values for resource comment and label:

  public void create() {
    Resource r = getInfo();
    r.setResourceComment(&quot;The ontology defining the most general concepts dealing with light sources and their control.&quot;);
    r.setResourceLabel(&quot;Lighting&quot;);

Can these properties be edited in the model?

Regards

--

From: _this issue has been automatically imported from our old issue tracker_

**Comment by cstockloew** [comment]
_Original author: Marina de la Fuente (2012-02-27 11:45:22)_

Some imports are also included automaticly:

addImport(ServiceBusOntology.NAMESPACE);
addImport(LocationOntology.NAMESPACE);

And these imports may not be needed.

Regards

**Comment by cstockloew** [comment]
_Original author: @erlendstav (2012-02-27 14:38:50)_

The UML comment text at the top level of the model (which is generated by the Ontology Wizard) has been intended for the purpose of setting this comment. The code has now been corrected to use this comment.
The resource label is set to the name of the ontology - i.e. the name of the rool element of the UML model.

The comment about imports I have added to another import-related bug.
